ui/package.json

```diff
@@ -36,7 +36,7 @@
     "@types/react-dom": "^17.0.2",
     "@typescript-eslint/eslint-plugin": "^5.36.0",
     "@typescript-eslint/parser": "^5.36.0",
-    "eslint": "~9.6.0",
+    "eslint": "~8.22.0",
```

Comment:
Is this change intentional in a PR that seems only about updating the changelog?

Reply:
Oops, nope, a left over file, too many context switching. Thx